### PR TITLE
Update memory-balloon design page

### PR DESCRIPTION
**Pull request by jtokar**
Changes proposed in this pull request:

- Fixed formatting.
- Changed some of the descriptions to match the actual code. 
- Fixed image link.

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @jtokar
This pull request needs review by: @MarSik @doron-fediuck

### CODE REVIEW

File: source/documentation/sla/memory-balloon.html.md [hunk 1]
```diff
@@ -55,28 +53,28 @@ oVirt's MOM sub-project integration[1](/wiki/Features/MomIntegration).
...
-      'device':'memballoon ',
-      'model': 'virtio' },     <--- 'none' may/should (need to check!) be used to disable the device.
+        {'type': 'balloon ',
+        'device':'memballoon ',
+        'model': 'virtio' },     <--- 'none' may/should be used to disable the device.
 
 ## Rest API
 
-*   A simple flag hould be added in the relevant actions.
+*   A simple flag should be added in the relevant actions.
```
Comment (MarSik) [inline source/documentation/sla/memory-balloon.html.md]: This is obsolete now. We should provide a links to:

- http://ovirt.github.io/ovirt-engine-api-model/4.1/#types/memory_policy
- http://ovirt.github.io/ovirt-engine-api-model/4.1/#types/cluster

**Comment by mykaul** [comment]
@jtokar - what is the latest here? Wast it reviewed and approved?

**Comment by mykaul** [comment]
@JohnMarksRH - please review this PR.

**Comment by jtokar** [comment]
@mykaul this needs review from @MarSik.

File: source/documentation/sla/memory-balloon.html.md [hunk 2]
```diff
@@ -19,16 +19,16 @@ on VM needs in a dynamic way. In this way it's possible to create memory over co
...
 ## Engine core
 
 *   **Backend-ui/rest parts**
-    -   A new VM attribute - enableBalloon - will be added VM & VmStatic BE to support this feature.
+    -   A new VM attribute - balloonEnabled - will be added VM & VmStatic BE to support this feature.
         -   The attribute is not persistant in vm_static table and clients should call the IsBalloonEnabledQuery to get the value from vm_device.
         -   Default value should be True.
-        -   This attribute should be handle in addVmCommand(s)
-    -   Add a device (check if add with model 'none' is needed)
+        -   This attribute should be handled in addVmCommand(s)
```
Comment (JohnMarksRH) [inline source/documentation/sla/memory-balloon.html.md]: This is OK.

**Pull request merged**